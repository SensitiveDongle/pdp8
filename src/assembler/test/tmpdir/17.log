Memory reference instruction operand must be constant or symbol
*** 1 errors in assembly
