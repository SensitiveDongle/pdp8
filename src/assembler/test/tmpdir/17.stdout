                  1: !@#$%^&*(){, 65
                  2: 
  0x000: 0x000    3: program, 0
  0x001: 0xE80    4: 		CLA
  0x002: 0x200    5: 		TAD !@#$%^&*()
  0x003: 0xC20    6: 		IOT 4 0
  0x004: 0xF02    7: 		HLT
                  8: 
                  9: END program
